src/macros.rst

```diff
@@ -980,49 +980,25 @@ When transcribing a metavariable
 A metavariable indication in a macro repetition in transcription shall be
 transcribed to the matched tokens in order,
 
-:dp:`fls_u2lq0lr12kdt`
```

Comment:
There is a bunch of nonsense on lines 976 and 977 which should be removed (our attempts to formulate a rule).